web/crux/src/app/node/node.message.ts

```diff
@@ -0,0 +1,87 @@
+import { Type } from 'class-transformer'
+import { IsDate } from 'class-validator'
+import { NodeConnectionStatus } from '../shared/shared.dto'
+
+export const WS_TYPE_NODE_EVENT = 'event'
+export class NodeEventMessage {
+  id: string
+
+  status: NodeConnectionStatus
+
+  address?: string
```

Comment:
Can we need to add `@isOptional()` here?

Reply:
I am not quite sure yet if the validator runs on the ws messages, probably we'll have to tweak it a little, but yeah we could add it.